### DISCUSSION TITLE
Null constraint violation on the fields: (`id`)

### DISCUSSION THREAD

**Discussion by ogldev** [prisma/prisma#18184] [Q&A]
Hello all! I have created a schema myself with using `id          String     @id @unique @default(cuid())` default in SvelteKit. When I'm using prisma.create, it asks me to enter an ID and gives error `Null constraint violation on the fields: (`id`)`. If I enter an ID in inputs the ID I give but I want it to be auto generated. Any help?

**Comment by ludralph** [comment]
Hi @Gyunbie 👋 

Can you please share your schema file?

**Reply by ludralph** [reply]
Hi @Gyunbie 

It would be really helpful if you could provide the additional information that we had asked about last time. If your problem is resolved by now or somehow doesn't feel relevant anymore, you can simply close the Discussion as well.

**Reply by ludralph** [reply]
Hi @Gyunbie, I'll be closing this discussion since there has been no response from you.
